### PR TITLE
few improvements

### PR DESCRIPTION
**Pull request by Xilinxx**
* remove accounts with incorrect password from input file with the [-d] argument
* have a connect [-c] argument (Sometimes you just wish to use the script to connect to the first working account)
* no duplicates in output file
* input file email:passw can have more characters behind the passw (break the pasw at the first < space >)
* README updated

### CODE REVIEW

File: nord-checker.py [hunk 1]
```diff
@@ -50,7 +65,12 @@ def check_login(email: str, password: str) -> Union['False', None, str]:
     )
 
     if not login_result.returncode == 0:
-        # Failed to login
+        # Failed to login - remove account data when password is incorrect
+        if (login_result.stdout.find('password is not correct') != -1):
+            if is_tool('sed'):
+                cmd = ['sed', '-i', '/{e}/d'.format(e=email), infile]
+                subprocess.run(cmd, capture_output=False)
```
Comment (behnambm) [inline nord-checker.py:72]: Remove this section. The script should not make any change in user owned files.

File: nord-checker.py [hunk 2]
```diff
@@ -88,14 +107,13 @@ def read_file(args) -> None:
         for line in f:
             if not line.strip():  # ignore empty lines in file
                 continue
-
+            account = line.split(' | ')
```
Comment (behnambm) [inline nord-checker.py:110]: the only separator is colon `:`. The pipe symbol makes it complicated to understand

File: nord-checker.py [hunk 3]
```diff
@@ -36,10 +36,25 @@ def read_arguments():
...
+        '-c',
+        '--connect',
+        help='login & connect',
+        action='store_true'
+    )
+
+
     return parser.parse_args()
 
+def is_tool(name):
```
Comment (behnambm) [inline nord-checker.py:49]: Use a better name for this function

File: nord-checker.py [hunk 4]
```diff
@@ -113,37 +131,63 @@ def read_file(args) -> None:
...
+        else:
+            with open(file_path, "a") as output_file:
+                output_file.write(entry + "\r\n")
 
 
 if __name__ == "__main__":
     args = read_arguments()
 
+    if is_tool('nordvpn'):
+        print(G+"\n...Nordvpn is installed!"+E)
```
Comment (behnambm) [inline nord-checker.py:186]: No need to print out this message. Show a message only when the `nordvpn` package is not installed.

File: README.md [hunk 5]
```diff
@@ -17,17 +17,35 @@ A simple Python script to check if NordVPN accounts listed in a file are valid.
...
 
+```
+$ python nord-checker.py --file succes.txt --output /dev/null -c
+```
+
+
+- Unique successful entries are appended to the specified output file.
+- The faulty entries will be removed from the input_file (requires sed)
+
+# Best Practise
```
Comment (behnambm) [inline README.md:39]: fix typo

**Review by behnambm — COMMENTED** [review]
At the end it would be great to reformat the source code with [black](https://black.readthedocs.io/en/stable/)

File: nord-checker.py (hunk 2 above)
Comment (Xilinxx) [reply nord-checker.py:110]: there are input lines looking like this
`gmail.com:Germany2016! |  Expires at: 2022/10/22`
Then you need to split at '|'or the password will be taken upto the last character.
Alternatively a <space> would be the best split.

File: nord-checker.py (hunk 1 above)
Comment (Xilinxx) [reply nord-checker.py:72]: Why would you want to keep accounts with a wrong password and have to check them again on next run?

File: nord-checker.py (hunk 4 above)
Comment (Xilinxx) [reply nord-checker.py:186]: Agreed

**Comment by Xilinxx** [comment]
> At the end it would be great to reformat the source code with [black](https://black.readthedocs.io/en/stable/)

Will do! never used black before :)

File: nord-checker.py (hunk 1 above)
Comment (behnambm) [reply nord-checker.py]: the argument can be a good approach.

File: nord-checker.py (hunk 2 above)
Comment (behnambm) [reply nord-checker.py]: No!

The script can not use anything other than `:` to split the `user:pass`

clearing the input `user:pass` is on user

**Comment by behnambm** [comment]
Thanks for spending time on this project I'm a little busy these days but I will review your PR ASAP.

**Pull request merged**